zenoh_cpp_vendor/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(ZENOHC_CARGO_FLAGS "--no-default-features$<SEMICOLON>--features=shared-memor
 # - https://github.com/eclipse-zenoh/zenoh-c/pull/620 (fix ze_querying_subscriber_get API to query newly discovered publishers)
```

Comment:
Can we delete the comments above and replace it with a single comment referencing the PR that fixed the build issue? The old issues should be outdated now that we've upgraded to 1.0